app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Why set to false? minifyEnabled is here to obfuscate the code. Proguard rules are provided so setting minifyEnabled to true won't cause a runtime crash